hosts_example.yml

```diff
@@ -43,6 +43,7 @@ all:
     ## As well as the signed certificate path and the key for that certificate for each host.
     ## These will need to be set for the correct host
     # ssl_custom_certs: true
+    # ssl_remote_certs: true # if you set this to true, ansible will resolve the paths on each kafka-broker to find key, crt and ca file
```

Comment:
Can you change this var to `ssl_custom_certs_remote_src`
And change the description to `set to true if key crt and ca file already on hosts, file paths still must be set'